library/psa_crypto.c

```diff
@@ -165,26 +170,36 @@ psa_status_t mbedtls_to_psa_error(int ret)
             return PSA_ERROR_INSUFFICIENT_MEMORY;
         case MBEDTLS_ERR_ASN1_BUF_TOO_SMALL:
             return PSA_ERROR_BUFFER_TOO_SMALL;
+#endif
 
+#if defined(PSA_WANT_KEY_TYPE_CAMELLIA)
 #if defined(MBEDTLS_ERR_CAMELLIA_BAD_INPUT_DATA)
```

Comment:
I think this guard should be removed now the new guard has been added

Reply:
It's still useful for `MBEDTLS_ERR_CAMELLIA_INVALID_INPUT_LENGTH` ?

Reply:
I'm not sure what you mean. I'm suggesting removing the `#if defined(MBEDTLS_ERR_CAMELLIA_BAD_INPUT_DATA)`-`#endif` pair now that we've added `#if defined(PSA_WANT_KEY_TYPE_CAMELLIA)`-`#endif`

Reply:
(and since `MBEDTLS_ERR_CAMELLIA_BAD_INPUT_DATA` and `MBEDTLS_ERR_CAMELLIA_INVALID_INPUT_LENGTH` are defined next to each other, we can't have one defined without the other)